app/views/government_ruleset/new.html.erb

```diff
@@ -3,5 +3,5 @@
     <h1 class="govuk-heading-xl">Check your OpenAPI documents correspond with UK government API linting rules.</h1>
   </div>
 
-  <%= render "form" %>
+  <%= render "government_ruleset/form" %>
```

Comment:
If new is rendered via a redirect rather than a render in another action the location of neighbouring files becomes simpler and this level of specificity becomes unnecessary. In other words `render "form"` is no longer problematic.